Dockerfile.rails-next

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && \
 ENV MALLOC_CONF="narenas:2,background_thread:true,thp:never,dirty_decay_ms:1000,muzzy_decay_ms:0"
 ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
 
+
```

Comment:
Nitpicky: we don't need this extra line in the Dockerfile.rails-next file. (This will minimize files changed on PR)